ec2/sc-ec2-linux-jumpcloud-notebook.yaml

```diff
@@ -409,7 +409,13 @@ Resources:
       TargetGroupArn: !Ref EC2TargetGroup
       ListenerArn: !ImportValue
         'Fn::Sub': '${AWS::Region}-alb-notebook-access-ALBListenerARN'
-
+  TagInsance:
```

Comment:
please correct the `TagInsance` misspelling -- occurs in all the additions.

Reply:
fixed